test/consistent-package-json.test.ts

```diff
@@ -0,0 +1,142 @@
+import {dirname, join, relative, resolve} from 'path';
+
+import {readJSONSync} from 'fs-extra';
+import glob from 'glob';
+
+const KNOWN_TEMPLATE_KEYS = [
+  'author',
+  'bugs',
+  'dependencies',
+  'description',
+  'files',
+  'homepage',
+  'license',
+  'main',
+  'name',
+  'publishConfig',
+  'repository',
+  'scripts',
+  'sideEffects',
+  'types',
+  'version',
+];
+
+const ROOT_PATH = resolve(__dirname, '..');
+const packageJSONTemplatePath = join('templates', 'package.hbs.json');
+const packageJSONTemplate = readJSONSync(
+  join(ROOT_PATH, packageJSONTemplatePath),
+);
+
+// eslint-disable-next-line jest/valid-describe
```

Comment:
I had to add this cause the linter is trying to check that the first argument to `describe` is a `string`, which in this case it is, it's just dynamic and coming from a variable. The linter can't figure that out though, so it fails.

Looks like [others](https://github.com/jest-community/eslint-plugin-jest/issues/203) have run into this issue before.

Reply:
This is just going to be `templates/package.hbs.json`, no?  I'd prefer just using that.